### DISCUSSION TITLE
Enable LSM permanently

### DISCUSSION THREAD

**Discussion by tdnguyenND** [cosmos/community#75] [Chain Builders (Cosmos SDK)] [answered]
Currently gaiad is running with cosmos-sdk@v0.47.10-ics-lsm, which enable LSM module. But the fact that other chains are usually forking the release version of cosmos sdk and maintain it themself, and the LSM module are not enabled in those chains.
Enable LSM in the release version will make the feature more popular, and more ideas for developing ecosystem will be possible

**Comment by tac0turtle** [comment, marked as answer]
We plan on making a different version of LSM. The LSM design in the sdk was not upstreamed after many users asked us not to do it. The current design also has some flaws in our opinion that we plan on working on in the next quarter

**Reply by tdnguyenND** [reply]
Hi @tac0turtle, is there any way I can track your plan of developing new LSM?
